lua/legendary/data/state.lua

```diff
@@ -9,5 +9,7 @@ local M = {}
 M.items = ItemList:create()
 M.most_recent_item = nil
 M.most_recent_filters = nil
+-- most recent per item group (added after v2.13.5)
```

Comment:
```suggestion
```

Reply:
Isn't it better to keep track of features added?

Reply:
The plugin uses semantic versioning. It will appear in the release notes.